src/apiCalls.js

```diff
@@ -0,0 +1,10 @@
+export const getAllMoviesData = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+}
+
+export const getSingleMovieData = (id) => {
+  const apiLink = 'https://rancid-tomatillos.herokuapp.com/api/v2//movies/'
+  return fetch(`${apiLink}${id}`)
```

Comment:
I bet we could refactor this to use only one function instead of two! But we can leave that for much later after we get through everything else! :)